lib/accounts/README.md

```diff
@@ -20,7 +20,9 @@ in place rebuild your project to generate stack targets for your account.
 # TARGET AUTHENTICATION
 
 **DO NOT UPDATE THE target_auth_template.json FILE DIRECTLY**
-The **target_auth_template.json** is a template json file for setting up your own identity provider (IdP) for tile server authentication. Copy this file into the `lib/accounts` directory with the name **target_auth.json** and configure it there. Once you have your configuration in place, ensure you have added `enableAuths: true` to `target_account.json`. Then, rebuild the project and deploy it.
+The **target_auth_template.json** is a template json file for setting up your own identity provider (IdP) for tile server authentication. 
```

Comment:
This is my idea, remove the template file. Then update the section to be something like this:


Below is a template json file for setting up your own identity provider (IdP) for tile server authentication. Copy this section into the `lib/accounts/target_account.json`. Once you have your configuration in place, rebuild the project and deploy it.

```
auth: {
    "clientId": "<your client id IdP>",
    "clientSecret": "<your client IdP secret>",
    "authority": "<your issuer IdP url>",
    "certificateArn": "<your certificate arn>",
    "domainName": "<your domain name>",
}
```

Reply:
done